finalPrep/modZeroHero.js

```diff
@@ -1,52 +1,79 @@
 // Challenge - See if you can follow the instructions and complete the exercise in under 30 minutes!
 
 // Declare two variables - heroName AND specialAbility - set to strings
+var heroName = "Batman";
+var specialAbility = "just being rich";
 
 // Declare two variables - greeting AND catchphrase
 //   greeting should be assigned to a string that uses concatenation to include the heroName
 //   catchphrase should be assigned to a string that uses interpolation to include the specialAbility
+var greeeting = ("I'm " + heroName + "!");
+var catchphrase = (`I'll save Gotham by ${specialAbility}.`);
+
 
 // Declare two variables - power AND energy - set to integers
+var power = 45;
+var energy = 80;
 
 // Declare two variables - fullPower AND fullEnergy
 //   fullPower should multiply your current power by 500
 //   fullEnergy should add 150 to your current energy
-
+var fullPower = (power * 500);
+var fullEnergy = (power + 150);
 // Declare two variables - isHuman and identityConcealed - assigned to booleans
-
+var isHuman = true;
+var identityConcealed = true;
 
 // Declare two variables - archEnemies AND sidekicks
 //   archEnemies should be an array of at least 3 different enemy strings
 //   sidekicks should be an array of at least 3 different sidekick strings
+var archEnemies = ["The Joker", "Poison Ivy", "Dr. Freeze"];
+var sidekicks = ["Robin", "CatWoman", "Alfred"];
 
 // Print the first sidekick to your console
-
+console.log(sidekicks[0]);
 // Print the last archEnemy to the console
-
+console.log(archEnemies[2]);
 // Write some code to add a new archEnemy to the archEnemies array
+archEnemies.push("The Penguin")
 
 // Print the archEnemies array to console to ensure you added a new archEnemy
+console.log(archEnemies);
 
 // Remove the first sidekick from the sidekicks array
+sidekicks.shift()
 
 // Print the sidekicks array to console to ensure you added a new sidekick
 
+console.log(sidekicks);
+
 // Create a function called assessSituation that takes three arguments - dangerLevel, saveTheDay, badExcuse
 //   - dangerLevel should be an integer
-//   - saveTheDay should be a string a hero would say once they save the day 
+//   - saveTheDay should be a string a hero would say once they save the day
 //   - badExcuse should be a string a hero would say if they are too afraid of the dangerLevel
+  function assessSituation(dangerLevel, saveTheDay, badExcuse) {
+    if (dangerLevel > 50) {
+        console.log(badExcuse);
+    } else if (10 <= dangerLevel && dangerLevel <= 50) {
```

Comment:
You can think about using the condition with your most complex logic in the `else` block.